src/main/java/org/matsim/episim/model/InfectionModelWithSeasonality.java

```diff
@@ -82,6 +82,10 @@ static double interpolateOutdoorFraction(EpisimConfigGroup episimConfig, int ite
 	static double getIndoorOutdoorFactor(double outdoorFraction, SplittableRandom rnd, EpisimConfigGroup.InfectionParams act1, EpisimConfigGroup.InfectionParams act2) {
 
 		if (!act1.isSeasonal() && !act2.isSeasonal()) return 1.;
+		
+		if (!act1.getContainerName().contains("leisure") && !act2.getContainerName().contains("leisure")) {
+			outdoorFraction = outdoorFraction * 0.5;
```

Comment:
Why is there now this hard-coded factor here ?

Reply:
In the default scenario, every activity is now seasonal and leisure reduced further as it seems.

Reply:
This is a change by @smuellervsp. I'm not sure if it makes sense that it is hardcoded...